lib/topic.ts

```diff
@@ -5,8 +5,7 @@ export const navigateToKafkaTopicsList = async function (page: Page, kafkaName:
   await expect(page.getByText(kafkaName)).toHaveCount(1);
   await waitForKafkaReady(page, kafkaName);
   await page.locator('a', { hasText: kafkaName }).click();
-  // data-testid=pageKafka-tabTopics
-  await page.locator('button', { hasText: 'Topics' }).click();
+  await page.getByTestId('pageKafka-tabTopics').click();
```

Comment:
Was the original selector not working reliably? I thought that removing the usage of `data-testid` was on purpose to have a unified approach.

Reply:
This one is more robust @mmusil

Reply:
in which way it is more robust? Someone can easily remove data-testid from code

Reply:
Can I resolve this conversation? @Frawless @mmusil